docs/apis/common/index.md

```diff
@@ -1350,3 +1350,397 @@ You may specify program arguments before the job is executed. The plan visualiza
+- `metrics.scope.jm.job`
+  - Default: \<host>.jobmanager.\<job_name>
+  - Applied to all metrics that were scoped to a jobmanager and job.
+- `metrics.scope.tm`
+  - Default: \<host>.taskmanager.\<tm_id>
+  - Applied to all metrics that were scoped to a taskmanager.
+- `metrics.scope.tm.job`
+  - Default: \<host>.taskmanager.\<tm_id>.\<job_name>
+  - Applied to all metrics that were scoped to a taskmanager and job.
+- `metrics.scope.tm.task`
+  - Default: \<host>.taskmanager.\<tm_id>.\<job_name>.\<task_name>.\<subtask_index>
+   - Applied to all metrics that were scoped to a task.
+- `metrics.scope.tm.operator`
+  - Default: \<host>.taskmanager.\<tm_id>.\<job_name>.\<operator_name>.\<subtask_index>
+  - Applied to all metrics that were scoped to an operator.
+
+Note that for metrics for which multiple formats may apply (like jm and jm.job) the most specific format takes precedence,
+in this case jm.job.
+
+The hierarchical orders are as follows:
+
+jm < jm.job 
+
+tm < tm.job < tm.task < tm.operator
+
+This hierarchy also defines which variables may be accessed. The `tm.operator` format may contain variables for jobs, whereas `tm.job` may not contain
+variables for operators. There is no restriction on order of variables.
+
+There is no restriction on the order of variables.
+
+The default scope for operator metrics will result in a metric name akin to `localhost.taskmanager.1234.MyJob.MyOperator.0`
+
+If you also want to include the task name, but omit the taskmanager information you can specify the following format:
+
+`metrics.scope.tm.operator: \<host>.\<job_name>.\<task_name>.\<operator_name>.\<subtask_index>`
+
+This could create the name `localhost.MyJob.MySource_->_MyOperator.MyOperator.0`.
+
+The following is a list of all variables available:
+
+- JobManager: \<host>
+- TaskManager: \<host>, \<tm_id>
+- Job: \<job_id>, \<job_name>
+- Task: \<task_id>, \<task_name>, \<task_attempt_id>, \<task_attempt_num>, \<subtask_index>
+- Operator: \<operator_name>, \<subtask_index>
+
+### Reporter
+
+Metrics can be exposed to an external system by configuring a reporter in the `conf/flink-conf.yaml`.
+
+- `metrics.reporter.class`: The class of the reporter to use.
+  - Example: org.apache.flink.metrics.reporter.JMXReporter
+- `metrics.reporter.arguments`: A list of named parameters that are passed to the reporter.
+  - Example: --host localhost --port 9010
+- `metrics.reporter.interval`: The interval between reports.
+  - Example: 10 SECONDS
+
+You can write your own `Reporter` by implementing the `org.apache.flink.metrics.reporter.MetricReporter` interface.
+If the Reporter should send out reports regularly you have to implement the `Scheduled` interface as well.
+
+By default Flink uses JMX to expose metrics.
+All non-JMXReporters are not part of the distribution and have to be added to the classpath manually, either by putting the jar into /lib
+or including it in the job jar.
+
+Flink supports the following systems:
```

Comment:
I think the `:` and the following sub sections don't go well together. I think either removing the sentence or saying something like `The following sections list the supported reporters` is better